README.md

```diff
@@ -0,0 +1,43 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Decode Morse Message
+
+> In this project we learned how to use a module inside our class.
+
+
+## Getting Started
+
+> To successfully utilize this project first of all you need to clone or download the repository, Then open the terminal
+
+``` bash command
+$ irb
+```
+
+## Author
+
+👤 **Anasse LAARABI**
+
+- GitHub: [@a-laarabi](https://github.com/a-laarabi)
+- Twitter: [@LusindisoNt](https://twitter.com/AnasseLaarabi)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/a-laarabi/)
+
+👤 **Tuyishimire Eric**
+
+- GitHub: [@TuyishimireEric](https://github.com/TuyishimireEric)
+- LinkedIn: [@TuyishimireEric](https://www.linkedin.com/in/TuyishimireEric/)
+- E-mail: <a href="mailto:tuyishimireericc@gmail.com">tuyishimireericc@gmail.com</a>
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/TuyishimireEric/Enumerable/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] I suggest customizing your README file to the milestone for this project so please add the information and the authors info so kindly edit it and send a PR again 📌💥✨ The README is very important to showcase your project 🤗

Reply:
# Thank you @Reem-lab for reviewing our project

We had forgotten to customizing our README file to the milestone for this project thank you for your suggestion